bir/codec/SPEC.md

```
@@ -287,9 +287,33 @@ Terminators end basic blocks:
+- `CALL`: Function call. Payload is a call site followed by an optional result operand and continuation block.
+- `ASYNC_CALL`: Start action. Payload is a call site, an isolated flag, the future result operand, and continuation block.
+- `WAIT`: Single wait action
+- `ALTERNATE_WAIT`: Alternate wait action
+- `WAIT_ALL`: Multiple wait action
 - `RETURN`: Return from function
 
+### Call Data Format
+
+Calls and start actions contain a concrete call-data struct. Its kind enum selects the variant fields. Call data does not contain a result operand or continuation.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)